app/views/TaxCodeDetailsPreviousYearsView.scala.html

```diff
@@ -20,45 +20,54 @@
 @import uk.gov.hmrc.play.views.html.helpers._
 @import includes.ptaHeader
 
-@this(mainTemplate: MainTemplate)
+@this(main: MainTempleteNew)
 
 @(taxCodeViewModel: TaxCodeViewModelPreviousYears)(implicit request: Request[_], messages: Messages, user: controllers.auth.AuthedUser, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer, ec: scala.concurrent.ExecutionContext)
 
 
 @header = @{
     ptaHeader(
-    displayBackLink = true,
+    displayBackLink = false,
     mainHeadingText = taxCodeViewModel.mainHeading,
-    preHeadingText = taxCodeViewModel.preHeader)
+    preHeadingText = taxCodeViewModel.preHeader,
+    showGdsVersion = true
+    )
 }
 
-@mainTemplate(
+@main(
     title = taxCodeViewModel.title,
     authedUser = Some(user),
-    pageHeader = Some(header),
-    articleClasses = Some("full-width")){
+    backLinkContent=messages("tai.back-link.upper"),
+    backLinkUrl = Some("javascript:history.go(-1)"),
```

Comment:
should we use javascript or `Some(routes.WhatDoYouWantToDoController.whatDoYouWantToDoPage.url),
`?

Reply:
👍  Incorporated the change

Reply:
is it worth checking with ux to see if this is the page we want to be sending the user back to?

Reply:
Verified 👍